autobump-spring-boot/src/main/java/com/github/autobump/springboot/services/WebhookService.java

```diff
@@ -0,0 +1,76 @@
+package com.github.autobump.springboot.services;
+
+import com.github.autobump.core.model.SettingsRepository;
+import com.github.autobump.core.model.events.CommentCreatedEvent;
+import com.github.autobump.core.model.events.PrClosedEvent;
+import com.github.autobump.core.model.events.PushEvent;
+import com.github.autobump.core.model.usecases.CommentCreatedUseCase;
+import com.github.autobump.core.model.usecases.PullRequestClosedUseCase;
+import com.github.autobump.core.model.usecases.RebaseUseCase;
+import com.github.autobump.springboot.configuration.Autobumpconfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.net.URI;
+import java.util.Locale;
+
+@Service
+public class WebhookService {
+    private final SettingsRepository settingsRepository;
+    private final Autobumpconfig autobumpconfig;
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
Can't we use an annotated logger like slf4j? see also `if (logger.isWarnEnabled())` on line 70 which this particular logger seems to require

Reply:
no, this logger is the one used by spring, Slf4j is no longer on the classpath, like discussed in the morning standups.
secondly the `isWarnEnabled()` is required by PMD